src/main/java/internetshop/controllers/ProductsController.java

```diff
@@ -0,0 +1,27 @@
+package internetshop.controllers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import internetshop.lib.Injector;
+import internetshop.model.Product;
+import internetshop.service.ProductService;
+
+@WebServlet("/products/all")
+public class ProductsController extends HttpServlet {
+    private static final Injector injector =
```

Comment:
```suggestion
    private static final Injector INJECTOR =
```
аналогічно в інших класах